draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2208,6 +2234,44 @@ Steps:
 11. return (x, y)
 ~~~
 
+## Choosing a mapping function {#choosing-mapping}
+
+This section gives brief guidelines on choosing a mapping function
+for a given elliptic curve.
+Note that the suites given in {{suites}} are recommended mappings
+for the respective curves.
+
+If the target elliptic curve is a supersingular curve supported by either the
+Boneh-Franklin method ({{bfmap}}) or the Elligator 2 method for A == 0 ({{ell2a0}}),
+that mapping is the recommended one.
+
+Otherwise, if the target elliptic curve is a Montgomery curve ({{montgomery}}),
+the Elligator 2 method ({{elligator2}}) is recommended.
+Similarly, if the target elliptic curve is a twisted Edwards curve ({{twisted-edwards}}),
+the twisted Edwards Elligator 2 method ({{ell2edwards}}) is recommended.
+
+The remaining cases are Weierstrass curves.
+For curves supported by the Simplified SWU method ({{simple-swu}}),
+i.e., with A != 0 and B != 0, that mapping is the recommended one.
+Otherwise, the Simplified SWU method for AB == 0 ({{simple-swu-AB0}})
+is recommended if the goal is best performance, while
+the Shallue-van de Woestijne method ({{svdw}}) is recommended
+if the goal is simplicity of implementation.
+(This is because the Simplified SWU method for AB == 0 requires implementing
+an isogeny map in addition to the mapping function, while the Shallue-van de
+Woestijne method does not.)
+
+Alternatively, if one mapping function that works with any curve is desirable,
```

Comment:
I wonder if this paragraph is needed, or if it can be replaced with something a bit lighter, e.g., "Note that the Shallue-van de Woestijne method ({{svdw}}) works with any curve and may be used in cases where a specific curve is not targeted." I read this as sort of recommending this as a viable alternative, which I don't think is a goal. Rather, targeting specific curves seems best. @armfazh, what do you think?

Reply:
I agree, it would be better to say " use the Shallue-van de Woestijne mapping if none of the above mappings applies to your curve" .

Reply:
OK, I pushed an edit that tones this down as suggested.

Reply:
Yep, thanks!